tests/labs/lie/test_lie_tensor.py

```diff
@@ -118,3 +118,20 @@ def _to_torch(t):
         # Check class-version
         jac_c, out_c = _to_functional_fmt(getattr(c, f"j{op_name}")(*c_inputs))
         torch.testing.assert_close([jac1, out], [jac_c, out_c])
+
+
+@run_if_labs()
+def test_backward_works():
```

Comment:
Should we loop over all Lie group types beyond SE3?

Reply:
Since, I'll make a new PR to clean the one with the Lie group example I can just throw a look with SO3 in there, it's pretty easy.